trim_galore/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Docker container for TrimGalore
+
+FROM ubuntu:latest
```

Comment:
Using latest has some risks, because this changes every 6 months. We should stick with the `ubuntu:22.04` which is the latest LTS (long time support) version of it.